crates/components/poseidon-halo2/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "poseidon-halo2"
+authors = ["TLSNotary Team"]
+description = "An experimental implementation of Poseidon based on PSE's poseidon-gadget"
+categories = ["cryptography"]
+license = "MIT OR Apache-2.0"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+name = "poseidon_halo2"
+
+[dependencies]
+ff = "0.13"
+group = "0.13"
```

Comment:
nit: i think we're adopting the format of always using `{ version = "0.17" }` in the project?